ui/app/scripts/task/controllers/definitions.js

```diff
@@ -90,6 +118,31 @@ define(['model/pageable'], function (Pageable) {
       $scope.bulkDefineTasks = function() {
         $state.go('home.tasks.bulkDefineTasks');
       };
+      $scope.isComposedTask = function(item) {
+        return item && angular.isString(item.dslText) && item.dslText.startsWith('composed-task-runner ');
```

Comment:
The name `composed-task-runner` isn't a fixed name for all the composed task and one can define custom name for this via dataflow property. Hence, we need a better approach to decide if the task is composed or not.